jscomp/test/ari_regress_test.js

```diff
@@ -82,4 +80,4 @@ var suites = /* :: */[
 
 Mt.from_pair_suites("Ari_regress_test", suites);
 
-/* x Not a pure module */
+/* partial_arg Not a pure module */
```

Comment:
Still true?

Reply:
What do you mean here, partial_arg is a function application where a side effect could happen?

Reply:
Oh perhaps it's clear if the test has an actual side effect.